data/cube/aggregate/LeafCountAggregator.js

```diff
@@ -0,0 +1,28 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2020 Extremely Heavy Industries Inc.
+ */
+
+import {Aggregator} from '@xh/hoist/data/cube/aggregate/Aggregator';
+
+export class LeafCountAggregator extends Aggregator {
+
+    aggregate(rows, fieldName) {
+        let count = 0;
+
+        this.forEachLeaf(rows, row => {
+            // TODO: Do we care about null values? Or are we counting Records? If we care about null, what about false?
+            // Example grouping: State -> Stores -> Employees... Employees has an 'isManager' flag... If I want to count the number of managers in a state, I'd want to exclude false as well as null
+            // But that doesn't help us in the case where you'd want the total number of employees in a state, which is a true count of Leaves,
+            count++;
```

Comment:
Need to discuss what the expectation of a COUNT aggregator really is.

In the use case in this PR it's more of the latter (all leaves regardless of value), but we could take the other approach since we are currently assigning a value of '1' to the count field for each leaf anyway.